sphinx/source/docs/user_guide/examples/extensions_ion_range_slider_template.tsx

```diff
@@ -0,0 +1,26 @@
+import * as DOM from "../../core/util/dom";
+
+interface RangeSliderProps {
+  id: string;
+  title?: string;
+}
+
+export default (props: RangeSliderProps): HTMLElement => {
+  let title, value;
+  if (props.title != null) {
+    if (props.title.length != 0) {
+      title = <label for={props.id}> {props.title}: </label>
+    }
+    value = <input type="text" id={props.id} readonly />
+  }
+
+  return (
+    <div class="bk-slider-parent">
+      {title}
+      {value}
+      <div class="bk-slider-horizontal">
+        <input type="text" class="slider" id={props.id}></input>
+      </div>
+    </div>
+  )
+}
```

Comment:
needing `.tsx` instead of `.eco` in any extensions also needs a migration note. It would also be good to at least include a mention of `.tsx` and maybe a link to some external docs for it in the "Extending Bokeh" users guide section.